challenges/security/application-security.md

```diff
@@ -0,0 +1,270 @@
+# Summary
+
+We use a synchronous, remote, one day hiring challenge for application
+security engineers. The challenge consists of the following sections:
+
+1. Kick off with the hiring manager. 45 minutes. 09:15 - 10:00 PST
+
+2. Peer review. You'll receive a brief Request for Discussion (RFD) and
+architecture diagram of a service. You will have 90 minutes to
+provide written PR feedback. 10:00 - 11:30 PST.
+
+3. After the peer review, section, there is a 30 minute break for lunch.
+
+4. Development. You'll receive prototype code for the service reviewed
+in part one. Building off this prototype codebase, we ask you develop
+some changes and open a pull request over 3 hours. 12:00 - 15:00 PST
+
+5. Attack & defense discussion.  This is a 60 minute video call with future
+peers where we ask you to show your depth of knowledge in hypothetically
+attacking a service, and defending against those attacks.
+You will also have a chance to ask us questions about working at Teleport.
+15:00 - 16:00 PST.
```

Comment:
What is this time for?

Reply:
This is when the attack and defense discussion will occur. Just like all the other sections have a time defined, so does this final step.